services/controllerhost/consumer.go

```diff
@@ -363,6 +363,85 @@ func fetchClassifyOpenCGExtents(context *Context, dstUUID string, cgUUID string,
 	return
 }
 
+// listConsummableExtents returns the list of extents for the destination that
+// are in open or sealed state and can be consumed by the given CG.
+func listConsummableExtents(context *Context, dstUUID string, cgUUID string,
```

Comment:
`Extents` are always prefixed with either `dst` or `cg` for clarity. 

consider renaming the signature as follows:

func findConsumableDstExtents(context, dstUUID, cgUUID string, openCGExtents map[], consumedCGExtents)